src/main/java/de/tum/in/www1/artemis/service/PrivacyStatementService.java

```diff
@@ -0,0 +1,120 @@
+package de.tum.in.www1.artemis.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.domain.PrivacyStatement;
+import de.tum.in.www1.artemis.domain.PrivacyStatementLanguage;
+import de.tum.in.www1.artemis.web.rest.errors.BadRequestAlertException;
+import de.tum.in.www1.artemis.web.rest.errors.InternalServerErrorException;
+
+@Service
+public class PrivacyStatementService {
+
+    private final Logger log = LoggerFactory.getLogger(PrivacyStatementService.class);
+
+    private static final String BASE_PATH = "privacy_statements";
```

Comment:
Currently, next to the `Artemis.war` executable three directories are used:
```
.
├── Artemis.war
├── config
│   ├── application-prod.yml
├── data
│   ├── course-archives
│   ├── repos
│   ├── repos-download
│   └── uploads
├── public
│   ├── content
|   |   ├── imprint.html
|   |   └── privacy_statement.html
│   └── images
└── templates
    └── jenkins
```
This is especially important when either (a) mounting those directories into containers, or (b) sharing those directories across systems for multi-instance setups like at TUM.

The data directories that are inside `data/` for me are configurable in `application*.yml` via
```
repo-clone-path
repo-download-clone-path
course-archives-path
submission-export-path
file-upload-path
```
This should be possible for any new directory storing additional data as well.